src/axom/quest/DistributedClosestPoint.hpp

```diff
@@ -682,10 +682,10 @@ class DistributedClosestPointImpl
       copy_components_to_interleaved(queryCoordsValues, xferDom["coords"]);
 
       xferDom["cp_index"].set_external(fields.fetch_existing("cp_index/values"));
-      xferDom["cp_domain_index"].set_external(fields.fetch_existing("cp_domain_index/values"));
       xferDom["cp_rank"].set_external(fields.fetch_existing("cp_rank/values"));
       copy_components_to_interleaved(fields.fetch_existing("cp_coords/values"),
                                      xferDom["cp_coords"]);
+      xferDom["cp_domain_index"].set_external(fields.fetch_existing("cp_domain_index/values"));
```

Comment:
Is this change necessary or incidental?
I'd guess the latter, but would be interested in hearing more if it is the former.

Reply:
It's incidental. I think I was just going back to a familiar code for some example, and I saw that moving that line of code would make the code trivially clearer.